print-pkg-name.py

```diff
@@ -0,0 +1,31 @@
+import pathlib
+import sys
+
+from packaging import utils
+
+
+def debug(msg: str):
+    print(f'::debug::{msg.title()}', file=sys.stderr)
+
+
+packages_dir = pathlib.Path(sys.argv[1]).resolve().absolute()
+
+wheel_file_names = [
+    f.name for f in packages_dir.iterdir() if f.suffix == '.whl'
+]
+sdist_file_names = [
+    f.name for f in packages_dir.iterdir() if f.suffix == '.gz'
+]
+
+# Parse the package name from the distribution files and print it. On error,
+# don't print anything.
+if wheel_file_names:
+    try:
+        print(utils.parse_wheel_filename(wheel_file_names[0])[0])
+    except utils.InvalidWheelFilename:
+        debug(f'Invalid wheel filename: {wheel_file_names[0]}')
+elif sdist_file_names:
+    try:
+        print(utils.parse_sdist_filename(sdist_file_names[0])[0])
+    except utils.InvalidSdistFilename:
+        debug(f'Invalid sdist filename: {sdist_file_names[0]}')
```

Comment:
There's a bit of nuance here: technically this action allows multiple projects to be released to at once, and there are almost certainly some users doing that with it. I _think_ we want the "magic link" behavior to gracefully handle this case, i.e. emit one link per each discovered project name in the dist listing.

Reply:
Fixed, this is what the log output looks like now, when uploading multiple projects:
<img width="1451" alt="image" src="https://github.com/user-attachments/assets/5e88b1b0-dc3b-4165-ae45-08aebe47508c">